### DISCUSSION TITLE
Possibility to change the text colorIOS for only one button

### DISCUSSION THREAD

**Discussion by Camposz** [mmazzarolo/react-native-modal-datetime-picker#690] [Ideas]
Implement the possibility to change text color of one of the buttons like this: 

![image](https://user-images.githubusercontent.com/66257593/215832100-9390d6eb-d085-45f0-b2a7-a840a6f5a93f.png)

**Comment by mmazzarolo** [comment]
You can already do that by providing your own `customCancelButtonIOS` and `customConfirmButtonIOS` components 👍
